types/adobe__pdfservices-node-sdk/operation/protect-pdf-operation.d.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * An operation that is used for securing PDF document with password(s).
+ * The password(s) is used for encrypting the PDF document and setting the restriction on certain features
+ * like printing, editing and copying in the PDF document.
+ *
+ * The supported algorithm for encrypting the PDF contents are listed here. The {@link PasswordProtectOptions.EncryptionAlgorithm} object can be used to set the
+ * encryption algorithm.
+ * <ul>
+ * <li>AES-128</li>
+ * <li>AES-256</li>
+ * </ul>
+ * For AES-128 encryption the password supports LATIN-I characters only.
+ * For AES-256 encryption the password supports Unicode character set.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        ProtectPDF = PDFServicesSdk.ProtectPDF,
+ *        protectPDFOptions = ProtectPDF.options,
+ *        permissions = protectPDFOptions.Permissions.createNew();
+ *
+ *  permissions.addPermission(protectPDFOptions.Permission.PRINT_LOW_QUALITY);
+ *  permissions.addPermission(protectPDFOptions.Permission.EDIT_DOCUMENT_ASSEMBLY);
+ *  permissions.addPermission(protectPDFOptions.Permission.COPY_CONTENT);
+ *
+ *  const options = new protectPDFOptions.PasswordProtectOptions.Builder()
+ *  		.setUserPassword("openpassword")
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/protect-pdf-operation.d.ts;
  Line: 31;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.